_posts/en/newsletters/2021-02-17-newsletter.md

```diff
@@ -0,0 +1,311 @@
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.12.1-beta.rc1][] is a release candidate for a maintenance
+  release of LND.  It fixes an edge case that could lead to accidental
+  channel closure and a bug that could cause some payments to fail
+  unnecessarily, plus some other minor improvements and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #20944][] adds a new `total_fee` field to the object returned
+  by the `getmempoolinfo` RPC and the `mempool/info` REST endpoint. `total_fee`
+  indicates the sum of the transaction fees for all transactions currently in the
+  mempool.
+
+- [LND #4909][] adds new `getmccfg` and `setmccfg` RPCs that can,
+  respectively, retrieve and temporarily change settings in LND's
+  *mission control* subsystem without restarting the daemon.  Mission
+  control uses information about past payment attempts to choose better
+  routes for subsequent payment attempts.
+
+- [Rust-Lightning #787][] ensures that channel closures due to error messages
+  only happen when the peer who sent the message is the channel's counterparty.
+  Previously, it was possible for malicious peers to force-close any channel if
+  they knew the channel id.
+
+- [BTCPay Server #2164][] redesigns the wallet setup wizard, walking the user
+  through setting up BTCPay's internal wallet and optionally integrating
+  it with the user's own remote software or hardware wallet.  This is
+  the start of other redesign work on BTCPay's interface.
+
+- [HWI #443][] adds support for signing multisig inputs with the
+  BitBox02 hardware wallet.
+
+- [Bitcoin Core #19145][] extends the `gettxoutsetinfo` RPC's
+  `hash_type` option to accept a new `muhash` parameter that will
+  produce the [MuHash3072][] digest of the UTXO set at the current block
+  height.  This is an alternative to producing the default SHA256 digest of
+  the UTXO set.  When run this way, MuHash has to process the same
+  amount as data as the SHA256 function, so there's not expected to be
+  any significant change in performance of the `gettxoutsetinfo` RPC,
+  which can take up to several minutes to return on slow single-board
+  computers.  However, a previously-calculated MuHash digest can have
+  elements added or removed from it relatively cheaply, so a future PR
+  is expected to quickly calculate a MuHash digest of the UTXO set at
+  each block, save those results in a simple database, and allow
+  returning them almost instantly upon demand.  This will also benefit
```

Comment:
```suggestion
  is expected to add a rolling MuHash object representing the UTXO set at
  each block height, save finalized digests for each height in an index, and allow
  returning them almost instantly upon demand.  This will also benefit
```